data/strategies/StrategyeCurveWETHSingleSided.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyeCurveWETHSingleSided",
+  "description": "Supplies WETH to the liquidity pool on Curve here to obtain eCRV tokens which it then puts into the v2 Curve sETH Pool yVault (yveCRV) to gain yield.",
```

Comment:
Why this is different from the Curve Voter Proxy strats? Doesn't it get the yield from CRV rewards?